app/apollo/resolvers/channel.js

```diff
@@ -589,8 +593,9 @@ const channelResolvers = {
           throw new NotFoundError(context.req.t('Channel uuid "{{channel_uuid}}" not found.', {'channel_uuid':channel_uuid}), context);
         }
 
-        // validate authorization on the channel
+        logger.info({req_id, user, org_id, channel_uuid, name, type}, `${queryName} validating`);
```

Comment:
This log line shouldn't be moved from its prior location just after the try{ line -- the lines between there and here are also 'validation'.